src/move_basic.cpp

```diff
@@ -799,8 +802,28 @@ bool MoveBasic::moveLinear(tf2::Transform& goalInDriving,
                                                             forwardRight);
         }
 
+	// Control to reference pose 
+	double K = 1.5;
+	double MOTOR_CONTROLLER_REACTIVITY = 0.02; // Changing this variable affects the reference pose accuracy, but it also regulates the deccelaration time - TODO: If higher or equal than 0.4 throws exception
```

Comment:
What do these parameters represent? Is is useful to have them as parameters to move_basic?

Reply:
K was changed to velMult, which represent the multiplier for the controlVelocity, which is a speed calculated from the distance to the reference point. The higher the velMult, more faster the robot converges to the reference pose.(Takes effect only when near to the goal, due to max. velocity limitation)

Reply:
MOTOR_CONTROLLER_REACTIVITY was changed to velThreshold, which is the lowest velocity input robot will get. If velocity lower than velThreshold, reference pose should be reached.

Reply:
Both of the values can be changed in order to get better accuracy, but at the same time to have a decent decceleration time of the robot to finally stop at the reference pose and mark the goal as reached.